pythonforandroid/recipes/pyleveldb/__init__.py

```diff
@@ -1,33 +1,13 @@
-from pythonforandroid.recipe import CompiledComponentsPythonRecipe
-from pythonforandroid.toolchain import current_directory
-import sh
+from pythonforandroid.recipe import CppCompiledComponentsPythonRecipe
 
 
-class PyLevelDBRecipe(CompiledComponentsPythonRecipe):
+class PyLevelDBRecipe(CppCompiledComponentsPythonRecipe):
```

Comment:
**As a side note:** That is my biggest concern, because we make dependant of `gnustl_shared` (which, as far as I know, is Deprecated in `ndk r18`)...so all recipes that depend on `gnustl_shared` will fail with newer ndks...but ,we have all relevant code concentrated in `CppCompiledComponentsPythonRecipe` :wink:

Related to: `Support for NDK 19` (#1613)
See also: https://developer.android.com/ndk/guides/cpp-support

Reply:
Good to know!